auto_editor/Fix1003Rule.py

```diff
@@ -17,5 +18,105 @@ def change_type(self) -> ChangeTypeEnum:
     def run_rule(self, project: StructuredProjectSource,
                  warning_first_line: int, warning_last_line: int, file_path: str) -> StructuredProjectSource:
         # Todo: add rule here
-        print("Add rule here")
+        first_time = True
+        warning_code, prefix = "", ""
+        tmp_dict = project.paths_to_lines
+        all_lines = tmp_dict[file_path]
+
+        for i in range(warning_last_line+1, len(all_lines)):
+            print("warning code:",all_lines[i].code)
+            if ";" not in all_lines[i].code:
+                now_code = self.strip_determin(first_time, all_lines[i])
+                warning_code = warning_code + now_code
+                warning_code = warning_code.replace("\n", "")
```

Comment:
Did you check that when the files are recreated from lineItems, this does not cause any issue? (The integration test temporarily creates a file and then deletes it again, so if you put a breakpoint before remove_destination() deletes the files, you can easily check this via integration tests, otherwise via script)

Reply:
”when the files are recreated from lineItems“？ which files？